combat-trainer.lic

```diff
@@ -3511,18 +3552,64 @@ class GameState
       return false
     end
 
-    find_cambrinth(@cambrinth, @stored_cambrinth, @cambrinth_cap)
-
     next_charge = @charges.pop
     echo("check_charging?: #{next_charge}") if $debug_mode_ct
```

Comment:
❓ I'm not a fan of assigning `next_charge`, echoing its value, then possibly changing it. What do you think about moving the echo or having a second echo?

Reply:
Alternatively, could `@charges.pop` be moved into `avtalia_charge`? I see a similar `.pop` on line 3579 with the recursive call.

Reply:
Can definitely do the former or something like it to make sure the debug remains consistent.  I'd rather avtalia_charge? continue to do absolutely nothing for anyone who isn't using it, just for the sake of sanity.

Reply:
I've kept the echo for check_charging? to mean 'how much is check_charging offloading onto your cambrinth?'  If avtalia_charge does anything on its own it reports how much it charged to what avtalia_cambrinth.  Anything left for check_charging will get echoed with the value it's actually using.